docs/hub/security-malware.md

```diff
@@ -0,0 +1,19 @@
+# Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
+
+Scanning is triggered at each commit or when you visit a repository page.
+
+Here is an [example view](https://huggingface.co/mcpotato/42-eicar-street/tree/main) of an infected file:
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/eicar-hub-tree-view.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/eicar-hub-tree-view-dark.png"/>
+</div>
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/eicar-hub-file-view.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/eicar-hub-file-view-dark.png"/>
+</div>
+
+_Note_: if your file has neither an ok nor infected badge, it could mean that it is either currently being scanned, waiting to be scanned, or that there was an error during the scan. It can take up to a few minutes to be scanned.
```

Comment:
I think we can use `doc-builder`'s special callouts for this type of side content now!

```suggestion
<Tip>
If your file has neither an ok nor infected badge, it could mean that it is either currently being scanned, waiting to be scanned, or that there was an error during the scan. It can take up to a few minutes to be scanned.
</Tip>
```

Reply:
Awesome, I didn't know about `<Tip>`! I tested this out with `doc-builder preview` and it worked fine as an `.md`, so hopefully it will also work when it's built.